.github/workflows/ssh-200.yml

```
@@ -0,0 +1,8 @@
+- name: executing remote ssh commands using password
```

actionlint (1.7.7)

workflow is sequence node but mapping node is expected

(syntax-check)